### ISSUE TITLE
about attribute  predict

### ISSUE THREAD

**Issue by hujiajia0401** [XiaoxiaoGuo/fashion-iq#5]
i don't see the image attribute dataset，can the attribute data find in the deepfashon data？please let me know。

**Comment by stankevichea** [comment]
> i don't see the image attribute dataset，can the attribute data find in the deepfashon data？please let me know。

Hi, did you find the attribute dataset? Thanks for answer

**Comment by nihirv** [comment]
I contacted the authors and the attribute data can be found here: https://github.com/hongwang600/image_tag_dataset

**Comment by zxh19976666** [comment]
> I contacted the authors and the attribute data can be found here: https://github.com/hongwang600/image_tag_dataset

sir how to download image by url？could you help me i‘m rookie....

**Comment by zxh19976666** [comment]
> I contacted the authors and the attribute data can be found here: https://github.com/hongwang600/image_tag_dataset

hey！How to divide the datasets into training set and test set，I have download all images，but how to divide it？

**Comment by dafeng-li** [comment]
> > I contacted the authors and the attribute data can be found here: https://github.com/hongwang600/image_tag_dataset
> 
> hey！How to divide the datasets into training set and test set，I have download all images，but how to divide it？

Hi, sir. Some url links are broken. How can I download the right pictures?
